HotelBooking.Tests/HotelBookedData.cs

```diff
@@ -0,0 +1,243 @@
+﻿using HotelBooking.Domain;
+namespace HotelBooking.Tests;
+
+public class HotelBookedData 
+{
+
+    public List<RoomType> TypesRoom = 
+    [
+        new(){ Id = 1,  NameRoom = "Luxary"},
```

Comment:
Здесь и далее по коду:
Для повышения читаемости кода лучше указывать тип при создании экземпляров объектов:
```suggestion
        new RoomType{ Id = 1,  NameRoom = "Luxury"},
```